.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,72 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "master" ]
+  schedule:
+    - cron: '37 21 * * 2'
```

Comment:
I see this is different from the [upstream template](https://github.com/github/codeql-action). If this is copied from one of our projects NB it's good to perturb the trigger day and time to reduce load spikes. (Both for the provider and for our reviewers!)

Reply:
I used GitHub's template to add this workflow. I didn't change any of its content.

Reply:
Presumedly fine then.